tests/schemas/invalid/test_invalid_answer_source_reference.json

```diff
@@ -0,0 +1,1487 @@
+                                "list_to_concatenate": [
+                                  {
+                                    "source": "answers",
+                                    "identifier": "first-name"
+                                  },
+                                  {
+                                    "source": "answers",
+                                    "identifier": "last-name"
+                                  }
+                                ]
+                              },
+                              "transform": "concatenate_list"
+                            },
+                            {
+                              "arguments": {
+                                "string_to_format": {
+                                  "source": "previous_transform"
+                                }
+                              },
+                              "transform": "format_possessive"
+                            }
+                          ]
+                        }
+                      ],
+                      "text": "What is <strong>{person_name_possessive}</strong> date of birth?"
+                    },
+                    "type": "General"
+                  },
+                  "when": {
+                    "==": [
+                      {
+                        "source": "answers",
+                        "identifier": "proxy-answer"
+                      },
+                      "Yes"
+                    ]
+                  }
+                }
+              ],
+              "type": "Question"
+            },
+            {
+              "id": "confirm-sex",
+              "question_variants": [
+                {
+                  "question": {
+                    "answers": [
+                      {
+                        "id": "confirm-sex-answer",
+                        "mandatory": true,
+                        "options": [
+                          {
+                            "label": "Yes, I am {sex-answer}",
+                            "value": "Yes, I am {sex-answer}"
+                          },
+                          {
+                            "label": "No, that is not my sex",
+                            "value": "No, that is not my sex"
+                          }
+                        ],
+                        "type": "Radio"
+                      }
+                    ],
+                    "id": "confirm-sex-question",
+                    "title": {
+                      "placeholders": [
+                        {
+                          "placeholder": "sex-answer",
+                          "transforms": [
+                            {
+                              "arguments": {
+                                "source": "answers",
+                                "identifier": "sex-answer"
+                              },
+                              "transform": "copy"
```

Comment:
Here also 😅, not sure why this has been added?

Reply:
Removed transform but also all the question variants so only one example was left here in the end.